src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4872,4 +4872,16 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureTuples" xml:space="preserve">
     <value>tuples</value>
   </data>
+  <data name="ERR_AmbiguousDeconstruct" xml:space="preserve">
+    <value>More than one Deconstruct instance or extension method was found for type '{0}'.</value>
+  </data>
+  <data name="ERR_DeconstructRequiresOutParams" xml:space="preserve">
+    <value>The Deconstruct method for type '{0}' must have only out parameters.</value>
```

Comment:
Similar for ERR_DeconstructWrongParams.

---
In reply to: [63824085](https://github.com/dotnet/roslyn/pull/11384#discussion_r63824085) [](ancestors = 63824085)